charter/index.html

```diff
@@ -208,6 +211,12 @@ <h3 id="out-of-scope">Out of Scope</h3>
             <p>The following features are out of scope, and will not be addressed by this Working group.</p>
 
           <ul class="out-of-scope">
+            <li>
+              Definition of a unique and exclusive solution for Personal Data Stores
+            </li>
```

Comment:
The spec is not called "Personal Data Stores Protocol", so naturally Solid Protocol is not parking PDS for all eternity.

There was no mention of PDS anywhere in the specs.

We also avoided the use of the informal / buzzword "pod" (personal online datastore) in Solid specs.

That's in addition to where I made a specific effort to avoid this situation:

* https://github.com/solid/solid-wg-charter/issues/3
* https://github.com/solid/solid-wg-charter/pull/25

So, as long as the charter is not pretending to be the solution for PDS, which I don't believe it does, I find this out of scope unnecessary.

Aside: I understand that you are trying to find ways to satisfy Members reviews - "business as usual". Hope this level of diligence making changes to the charter goes into each formal objection or change request raised by Members.

Reply:
> So, as long as the charter is not pretending to be the solution for PDS, which I don't believe it does, I find this out of scope unnecessary.

Whether the charter says it or not, some reviewers read it like that. So...

> Aside: I understand that you are trying to find ways to satisfy Members reviews - "business as usual".

you got it.

> Hope this level of diligence making changes to the charter goes into each formal objection or change request raised by Members.

Doing my best.